tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -26,8 +26,23 @@ using ::testing::ElementsAreArray;
 
 class BasePoolingOpModel : public SingleOpModel {
  public:
-  // TODO(ahentz): Also test different activation types, bias, padding types,
-  // stride values.
+  BasePoolingOpModel(const BuiltinOperator& type, const TensorData& input,
+                     int stride_width, int stride_height, int filter_width,
+                     int filter_height, const Padding& padding,
+                     const ActivationFunctionType& fused_activation_function,
+                     const TensorData& output) {
+    input_ = AddInput(input);
+    output_ = AddOutput(output);
+
+    SetBuiltinOp(type, BuiltinOptions_Pool2DOptions,
+                 CreatePool2DOptions(builder_, padding, filter_width,
```

Comment:
CreatePool2DOptions(builder_, padding, stride_width,
                                     stride_height, filter_width, filter_height,
                                     fused_activation_function)

Reply:
@haozha111 , added stride_width,stride_height as passed from the function.